### PR TITLE
Testing/skill page

### LINKED ISSUE #30: FE: End to End Testing in Cypress

### LINKED ISSUE #39: FE: Git Rebase

### PR DESCRIPTION
**Pull request by aominhlong**
## Description 
Add testing to the skill-page and add query name to FETCH_SKILLS in app

#### Where to start?
Start in the `skill-flow-cy.js` file

#### How to test?
In the project directory, you can run:

### `npm start`
Runs the app in the development mode.\
Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
The page will reload when you make changes.

### `npm run cypress`
-click 'login-flow' in the E2E specs section
-The test will automatically run.

#### Issues
Closes #39 
Closes #30 

#### Screenshots (if appropriate)
image

#### Type of Change:
- [] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (code is working and without bugs, formatting has been streamlined)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?
What steps have you taken to ensure the code is effective?
- [x] Have you used Dev Tools/ ran code in the console?
- [x] Have you ran the Cypress testing suite?

#### Checklist:
- [x] My code follows the style guidelines of this project.
- [x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- x[] I have performed a self-review of my own code.
- [x] I have added reviewers.

### CODE REVIEW

File: cypress/e2e/skill-flow.cy.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import { aliasQuery, aliasMutation, hasOperationName } from '../utils/graphql-test-utils'
+
+describe('Skill Page', () => {
+  beforeEach(() => {
+
+    cy.intercept('POST', 'https://pup-trainer-api.herokuapp.com/graphql', (req) => {
+      const { body } = req
+      aliasQuery(req, 'fetchUser')
+      if (hasOperationName(req, 'fetchUser')) {
```
Comment (Jlavarine) [inline cypress/e2e/skill-flow.cy.js:9]: Great job intercepting the query!

**Review by Jlavarine — APPROVED** [review]
Testing is important!

File: src/App.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const App = () => {
 
   const [ skills, setSkills ] = useState([])
     const FETCH_SKILLS = gql`
-    query {
+    query fetchSkills {
```
Comment (JoshMallery) [inline src/App.js:25]: glad we got the operational name added in there.

**Review by JoshMallery — APPROVED** [review]
Looks good, we got the test passing.

**Pull request merged**